.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
         # pre-commit's default_language_version, see
         # https://pre-commit.com/#top_level-default_language_version
         language_version: python3.10
-  - repo: https://github.com/pyCQA/isort
+  - repo: https://github.com/PyCQA/isort
     rev: 5.10.1
```

Comment:
I think you need to run `pre-commit autoupdate` since 5.10.1 won't install any more.

Reply:
How do you spell that?  I don't see `pre-commit` in my $PATH.  Is this related to the ups table or separate?

Reply:
Thanks for catching this, autoupdate seemed to work well:
```
$ pre-commit autoupdate
Updating https://github.com/pre-commit/pre-commit-hooks ... [INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
updating v4.3.0 -> v4.4.0.
Updating https://github.com/psf/black ... [INFO] Initializing environment for https://github.com/psf/black.
updating 22.3.0 -> 23.1.0.
Updating https://github.com/PyCQA/isort ... [INFO] Initializing environment for https://github.com/PyCQA/isort.
updating 5.10.1 -> 5.12.0.
Updating https://github.com/PyCQA/flake8 ... [INFO] Initializing environment for https://github.com/PyCQA/flake8.
updating 4.0.1 -> 6.0.0.
```

Reply:
PS - `pre-commit` for me appears in my $PATH after setting up the stack in my working shell.

Reply:
(you have to install `rubin-env-developer` to get it in your conda env if you aren't using a shared stack)